_posts/2022-04-20-unmap-me-project.markdown

```diff
@@ -61,6 +61,48 @@ Admin:
 
 ## Design Specifications
 
+### Data Model
+In general, there are roughly three main types of data:
+
+- Users
+- UnMaps
+- Locations
+
+#### User
+A user is a pretty basic, it has the following pieces of data:
+
+- A username
+- A name
+- An email address
+
+There will be additional details, like password hashes and what not, that will likely be delegated to AWS Cognito.
+
+#### UnMaps
+An UnMap represents a collection of locations that all share some common elements. They will need the following data:
+
+- A name
+- A slug (used for the url, i.e. is the slug is `kebab`, then the url would be `kebab.unmap.me`)
+- A description
```

Comment:
Wouldnt this be in the "Location Data Model" ? I can see why some types of locations wouldn't need a description.

Reply:
This is for the description of the `unmap`. As in..
> The Kebab UnMap is for tracking your favorite local Kebab shops!